manager/volume.go

```diff
@@ -149,6 +150,11 @@ func (m *VolumeManager) Create(name string, spec *longhorn.VolumeSpec, recurring
 		}
 	}
 
+	// restore backing image if needed
+	if err := m.restoreBackingImage(spec.BackingImage); err != nil {
+		return nil, errors.Wrapf(err, "failed to restore backing image %v when create volume %v", spec.BackingImage, v.Name)
+	}
+
```

Comment:
restore volume is actually create volume with FromBackup in the spec
in create volume function we check

1. if backing image exists
2. if backing image doesn't exist, check if backup backing image exists
3. create backing image by restore if backup backing image exists